client/src/styles/cms-overrides.scss

```diff
@@ -0,0 +1,8 @@
+.form-group--no-divider {
```

Comment:
Can you please add a comment saying why you're overriding something from the CMS? This looks very "global" and has no justification for why it exists.

Reply:
Comment added, not actually an override (to my knowledge)

Reply:
Oh, it's in a file called "CMS overrides"

So the classes we _do_ have target `.form--no-dividers .form-group` and `.form-field--no-divider` (in Form.scss from admin), is there a way you can use one of those? If not then assuming this would be PR'd as a third class into that module, could you use the same rules?